node/src/components/block_validator.rs

```diff
@@ -60,7 +60,7 @@ impl ProposedBlock<ClContext> {
 
     /// How many standard transactions?
     pub(crate) fn standard_count(&self) -> usize {
-        self.value().count(Some(TransactionCategory::Standard))
+        self.value().count(Some(TransactionCategory::Large))
     }
```

Comment:
It seems like the count checking logic is incomplete as it doesn't consider all categories (see here and the logic in fn validate_transaction_category_counts below).